spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -360,6 +360,18 @@
     end
   end
 
+  context ".build_relats" do
+    it "includes reflections from descendant classes of Vm" do
+      relats = MiqExpression.get_relats(Vm)
+      relats[:reflections][:cloud_tenant].should_not be_blank
+    end
+
+    it "includes reflections from descendant classes of Host" do
+      relats = MiqExpression.get_relats(Host)
+      relats[:reflections][:cloud_networks].should_not be_blank
+    end
```

Comment:
@gtanzillo Please add a negative test here using a class that does not respond to `include_descendant_classes_in_expressions?`